src/lora_plan.erl

```diff
@@ -233,13 +233,16 @@ up_to_down_datarate(Plan, Index, Offset) ->
         % io:format("up_to_down_datarate - DownIndex=~w~n", [DownIndex]),
         DownIndex
     catch
+        % Datarate Index is unknown or RFU
+        % Respond with RX2 default datarate
         _Class:_Reason:_Stacktrace ->
             lager:error(
-                "lora_plan=up_to_down_datarate bad index value Region=~p Index=~p Offset=~p~n", [
+                "lora_plan=up_to_down_datarate unknown datarate index Region=~p Index=~p Offset=~p~n",
+                [
                     Region, Index, Offset
                 ]
             ),
-            Index
+            Plan#channel_plan.rx2_datarate
```

Comment:
This return value is confusing to me.  Its former value and value from other return in same function (just above `catch`) are both an index, but this is datarate.

If types for these values are synonymous, just add a comment here saying so, and I'll approve this PR.

Otherwise, one of these return values seems incorrect.

Reply:
It is correct.

Throughout the library datarate atom, binary and index are all treated synonymously.  Further the datarate APIs on line 100 were carefully constructed to take any type, e.g. atom, binary or index and properly process them.  The resulting API basically does allow the exact type to be ignored mentally.  The code will always handle it correctly.

I knew this was an index based on the channel_plan record definition of integer.  An integer is always a datarate index.